docs/release-notes/highlights-2.11.0.asciidoc

```diff
@@ -1,6 +1,11 @@
 [[release-highlights-2.11.0]]
 == 2.11.0 release highlights
 
+[float]
+[id="{p}-2110-known-issues"]
+=== Known issues
+- The `resourceStatuses` field of the status subresource of the Stack Configuration Policy was removed which could cause errors in the operator logs during an upgrade.
```

Comment:
I'm unsure about this wording, and how we would prefer to word this. Comments appreciated.